torch/csrc/jit/passes/remove_mutation.cpp

```diff
@@ -248,6 +248,15 @@ bool MutationRemover::RemoveTensorMutation(Block* block) {
     }
 
     Value* mutated_value = node->inputs().at(0);
+    // If tranpose is inplace, then it should not be converted to its
+    // functional equivalent if the input tensor to it has more than
+    // one use.
```

Comment:
This isn't the condition for checking if `transpose_` can be unmutated safely.
It's simply to illustrate the example in the test case.